lib/sig0.js

```diff
@@ -81,22 +84,34 @@ sig0.makeKey = function makeKey(algorithm, priv) {
   return sig0.createKey(algorithm, pub);
 };
 
-sig0.createKey = function createKey(algorithm, publicKey) {
+sig0.createKey = function createKey(algorithm, publicKey, prefix) {
   assert((algorithm & 0xff) === algorithm);
   assert(Buffer.isBuffer(publicKey));
 
+  prefix = toName(prefix);
```

Comment:
The `prefix` is only useful for `PRIVATEDNS`, consumers of the function do not need to pass it unless it is that type. It also looks like it is only used from within the `if` block below. `toName` asserts that its input is a buffer/string, so this change may break code that uses `sig0.createKey` already.

Could you move the `prefix = toName(prefix)` inside of the `if` block below? This way consumers will only need to pass the `prefix` if they are using `PRIVATEDNS`.

Reply:
Fixed.